polaris-react/src/components/IndexTable/IndexTable.tsx

```diff
@@ -109,6 +109,7 @@ export interface IndexTableBaseProps {
   emptyState?: React.ReactNode;
   sort?: React.ReactNode;
   paginatedSelectAllActionText?: string;
+  defaultPaginatedSelectAllText?: string;
```

Comment:
Is there a reason why this should be prefixed with "default" and not just be `paginatedSelectAllText`?